core/task/schedulerstate.go

```diff
@@ -36,6 +36,7 @@ import (
 	"github.com/AliceO2Group/Control/common/logger/infologger"
 	"github.com/AliceO2Group/Control/core/controlcommands"
 	"github.com/AliceO2Group/Control/core/task/schedutil"
+	"github.com/gogo/protobuf/proto"
```

Comment:
[`gogo/protobuf`](https://github.com/gogo/protobuf) is deprecated though, can you try using a different one?

Reply:
i think this is the official one now: https://github.com/golang/protobuf

Reply:
no, it will not work, see previous build error.. but this package is used in mesos, so we are using it either way inherently, so I don't mind using it for cloning an object, that is already created by this package from mesos

Reply:
but I personally also don't mind using marshall and unmarshall because we are using it once per run and per offer

Reply:
ok, since mesos brings it anyway, let's keep gogo/protobuf, thanks for checking.